.github/workflows/playwright.yml

```diff
@@ -0,0 +1,29 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: lts/*
+    - name: Install dependencies
+      run: npm ci
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps chromium
+    - name: Run Playwright tests
+      # run: npx playwright test
+      # Only test on Chromium
+      run: npx playwright test --project=chromium
```

Comment:
What's the version that is used for testing?

Reply:
Each version of playwright is tied to the latest version of Chromium. So we will need to keep playwright up to date for this. The version is listed here:

https://github.com/microsoft/playwright?tab=readme-ov-file#documentation--api-reference

Reply:
Please add all of comments above to the code for the future maintenance.